jormungandr/src/rest/v1/logic.rs

```diff
@@ -91,11 +93,20 @@ pub async fn post_fragments(
         .map(|fragment| fragment.id().to_string())
         .collect();
     let mut msgbox = context.try_full()?.transaction_task.clone();
-    for fragment in fragments.into_iter() {
-        let msg = TransactionMsg::SendTransaction(FragmentOrigin::Rest, vec![fragment]);
-        msgbox.try_send(msg)?;
+    let (reply_handle, reply_future) = intercom::unary_reply();
+    let msg = TransactionMsg::SendTransactions {
+        origin: FragmentOrigin::Rest,
+        fragments,
+        fail_fast: true,
+        reply_handle,
+    };
+    msgbox.try_send(msg)?;
+    let reply = reply_future.await?;
+    if reply.is_error() {
+        Err(Error::Fragments(fragment_ids))
```

Comment:
... all this added summary serialization gets dropped at this point, right?

I think it's OK for a quick backport, though.

Reply:
This is not going to be used during fund4 anyways I think.